test/errorReports/too_long_line_both_sides_short.sol.log

```diff
@@ -0,0 +1,7 @@
+Warning: This is a pre-release compiler version, please do not use it in production.
+too_long_line_both_sides_short.sol:1:1: Warning: Source file does not specify required compiler version!
+contract C {
+^ (Relevant source part starts here and spans across multiple lines).
+too_long_line_both_sides_short.sol:2:15: Error: Identifier not found or not unique.
+   function f(announcementTypeXXXXXXXXXXXXXXXXXXX ... XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX Type,
+              ^-------------------------------------------------------------------------^
```

Comment:
I think this is probably out of scope for this change, but it would be great to have a matching 
` ^--------------------------------- ... ------------------------------------^` here